src/redux/pattern/pattern.types.js

```diff
@@ -0,0 +1,2 @@
+export const GET_PATTERNS = 'GET_PATTERNS';
+export const SET_PATTERNS = 'SET_PATTERNS';
```

Comment:
Add `LOADING_PATTERNS` type in case of slow connection.

Reply:
done